examples/feast/default_feature_store/feature_repo/example_repo.py

```diff
@@ -0,0 +1,130 @@
+# This is an example feature definition file
+
+from datetime import timedelta
+
+import pandas as pd
+from feast import Entity, FeatureService, FeatureView, Field, FileSource, PushSource, RequestSource
```

Comment:
Can we switch to doing:
```python
import feast
```
here please? Then it'll mean that all these classes will be prefixed by `feast.`. I think that will make it clearer to a reader when reading the code below, that it comes to feast. I know this is the example they provide, but I think that should help ensure that all the other examples it's also clear what's going on. Thoughts?